File: backend/Multibanca.Domain.Models/Multibanca/rectificatoria_legal_firma_alzante.cs
using Common.Domain.Models;

namespace Multibanca.Domain.Models.Multibanca
{
    public class rectificatoria_legal_firma_alzante : base_auditoria
    {
        public int id_rectificatoria_legal_firma_alzante { get; set; }
        public long id_expediente { get; set; }
        public int id_usuario_solicitante { get; set; }
        public DateTime? fecha_firma_alzante { get; set; }
        public string? observaciones { get; set; }
        public string? nombre_banco_alzante { get; set; }
    }
}

File: frontend/src/features/actividades/rectificatoria_legal_firma_alzante/models/rectificatoria_legal_firma_alzante.ts
export interface RectificatoriaLegalFirmaAlzante {
  id_rectificatoria_legal_firma_alzante: number;
  id_expediente: number;
  id_usuario_solicitante: number;
  fecha_firma_alzante: string | null;
  observaciones: string | null;
  nombre_banco_alzante: string | null;
  is_active: boolean;
  row_status: boolean;
  created_by: number;
  created_date: string;
  modified_by?: number | null;
  modified_date?: string | null;
}
